onnx/test/optimizer_test.py

```diff
@@ -99,6 +99,20 @@ def test_fuse_transpose_into_gemm(self):
         assert len(list(optimized_model.graph.node)) == 1
         assert optimized_model.graph.node[0].op_type == "Gemm"
 
+    def test_fuse_conv_add_into_bias(self):
+        conv = helper.make_node("Conv", ["X", "Y"], ["Z"])
```

Comment:
I would like to have another case, with broadcast enabled, our optimizer should not have any optimization

Reply:
I think our optimizer should only work with broadcast enabled and axis=1.
So why with broadcast enabled is unavailable?

Reply:
I mean we have some constraints on Conv's bias, it has to be a 1D tensor with size M (which is the feature number of W). So in case broadcasting case, this fusion cannot be conducted.

Reply:
Depends on above comment mentioned that we need conv shape information to check this.